### PR TITLE
Konveyor Charter first draft

### PR DESCRIPTION
**Pull request by jwmatthews**

### CODE REVIEW

**Review by rromannissen — APPROVED** [review]
LGTM! Thanks for putting this together John, it looks amazing.

**Review by savitharaghunathan — APPROVED** [review]
/lgtm
Thank you @jwmatthews :)

File: Charter.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Konveyor Charter
+
+## Overview
+Konveyor is an open-source application modernization platform that helps organizations safely and predictably modernize applications to new technologies, with an initial focus on accelerating the adoption of legacy applications to Kubernetes.
+
+The project’s design incorporates years of experience with consulting engagements successfully helping companies move existing applications to new technologies, such as Kubernetes.  The patterns and processes that led to successful, predictable, and safe modernization engagements were distilled into an [open-source methodology](https://github.com/konveyor/methodology).  This methodology is not required to be used with Konveyor but remains available to help guide others embarking on their own modernization journey who don’t have a preferred methodology.
```
[the remark this answers is not in the document]
Comment (djzager) [reply Charter.md:6]: nit

**Review by pranavgaikwad — APPROVED** [review]
LGTM on the content

**Review by shawn-hurley — APPROVED** [review]
I am also a plus one on this!

**Comment by asinghee** [comment]
Looks good @jwmatthews !

**Review by ibolton336 — APPROVED** [review]
Nice! LGTM

File: Charter.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  * Ease the transition to containerization and Kubernetes by assisting with the generation of artifacts for applications at scale
+  * Allow recipes or remediation workflows to be constructed and run to kickstart the work a Migrator needs to accomplish their modernization work.
+* Allow organizations to manage their application portfolio in a centralized fashion, and from there be able to classify applications and launch any kind of actions that might be related to them, such as analysis and assessments
+* Establish a pattern of continual enhancement of technology-specific information via leveraging the open source community to contribute knowledge and concerns related to modernizing applications to Kubernetes
+  * A  [user-group of subject matter experts](https://github.com/konveyor/community/tree/main/ug-migration-experience) has been established who are engaging in application modernization efforts on a regular basis.  The intent is to solicit use cases, problems, recommendations, and remediations from the challenges the members encounter and bring those learnings back into the larger community and platform.
```
[the remark this answers is not in the document]
Comment (jwmatthews) [reply Charter.md:38]: Thanks for noticing @aufi, I will be working to flesh some of the content out this week for this area in the community repo.

File: Charter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwmatthews) [reply Charter.md]: Thank you, I incorporated your suggestion.

**Pull request merged**